webui/react/src/components/TableActionBar.tsx

```diff
@@ -154,39 +144,43 @@ const TableActionBar: React.FC<Props> = ({
     useModal(ExperimentTensorBoardModal);
   const isMobile = useMobile();
   const { openToast } = useToast();
-  const experimentIds = useMemo(
-    () => Array.from(selectedExperimentIds.keys()),
-    [selectedExperimentIds],
-  );
+
+  const [experiments, setExperiments] = useState<Loadable<BulkExperimentItem>[]>([]);
+
+  const fetchExperimentsByIds = useCallback(async (selectedIds: number[]) => {
+    const response = await getExperiments({
+      experimentIdFilter: {
+        incl: selectedIds,
+      },
+    });
+    const loadedExperiments = response.experiments;
+    setExperiments(loadedExperiments.map((experiment) => Loaded(experiment)));
+  }, []);
```

Comment:
Two concerns about this approach.

One concern is that when `incl` has more than 143 ids (idk why 143 though),  `500 Internal Server Error` is returned. 143 isn't a big number so user probably encounter `500 internal server error` easily. 

https://github.com/determined-ai/determined/assets/109113616/cc79ff39-2812-4909-8f42-1dde0fd57b69

---

Furthermore, response data size would get fairly big enough if we can unlimitedly add `ids` in `incl`. The compressed experiment size from API response on my local is `1.7KB`. If i select 1000 experiments, it would be `1.7MB`. The actual users' experiment size could be bigger than `1.7KB`, so they probably see a performance issue. 

If we are optimistic about these issues for now, I'm happy to approve this PR. 
@corban-beaird what do you think?

Reply:
trying to mitigate these concerns by limiting how many experiments are fetched at a time